lib/rules/web/admin_console/4.2/common_ui_elements.xyaml

```diff
@@ -0,0 +1,301 @@
+create_resource_by_default_yaml:
+  elements:
+  - selector:
+      xpath: //button[@id="yaml-create"]
+    op: click
+  action:
+    if_element:
+      selector:
+        xpath: //a[text()='Edit YAML']
+    ref: click_edit_yaml_link
+  action: click_create_button
+click_edit_yaml_link:
+  elements:
+  - selector:
+        xpath: //a[text()='Edit YAML']
+    op: click
+click_create_button:
+  elements:
+  - selector:
+      xpath: //button[contains(text(),'Create')]
+    op: click
+click_cancel_button:
+  elements:
+  - selector:
+      xpath: //button[contains(text(),'Cancel')]
+    op: click
+click_button:
```

Comment:
`click_button_on_page` is changed to `click_button` with parameter `<text>` changed to `<button_text>` @SSshahan

Reply:
https://github.com/openshift/verification-tests/blob/master/lib/rules/web/admin_console/4.1/common_ui_elements.xyaml#L27

Reply:
Do we need update this rule in 4.1?